keyboards/keychron/k4/rgb/v2/readme.md

```diff
@@ -0,0 +1,29 @@
+# keychron_K4V2_RGB_ANSI
+
+
+*A 96% keyboard sold by keychron*
+
+* Keyboard Maintainer: [PythonDeployer](https://github.com/PythonDeployer) (big thanks to the SonixQMK discord)
+* Hardware Supported: *Keychron K4V2 RGB, MCU: SN32F248BF*
+* Hardware Availability: *https://www.keychron.com/products/keychron-k4-wireless-mechanical-keyboard-version-2*
+
+Make example for this keyboard (after setting up your build environment):
+
+    qmk compile -kb keychron/k4/rgb/v2/ -km ansi
+
+Make example for this keyboard (after setting up your build environment):
+
+    qmk compile -kb keychron/k4/rgb/v2/via -km ansi

```

Comment:
this kind of structure is a bit oldschool and dirty. The proper way to do it is to have via support in a via keymap.
with KC boards being both ANSI and ISO, it makes sense to treat them as variants. eg
`keychron/k4/{rgb,white}/{v1,v2}/{ansi,iso}/keymaps/{default,via}/keymap.c`